nautobot/ui/src/utils/url.js

```diff
@@ -2,3 +2,15 @@
 export function uiUrl(apiUrl) {
     return apiUrl.replace("/api/", "/");
 }
+
+/* Build url from app_label, model_name and optional pk */
+export function buildUrl(appLabel, modelName, pk = null, isPlugin = false) {
```

Comment:
Should this function account for the few `slugs` that still exist in the url after https://github.com/nautobot/nautobot/pull/3914?

Reply:
Not really, if a slug is passed in to the buildUrl it would still work as expected.
Do you have any in mind, just to verify